src/configure-webpack/index.js

```diff
@@ -19,7 +19,7 @@ import buildLoadersConfig from './loaders'
  * @return ready to use webpack configuration as an array
  */
 export default (saguiConfig = {}) => {
-  const { pages = [], libraries = [], ...sharedSaguiConfig } = saguiConfig
+  const { pages = [], libraries = [], chunks = {}, ...sharedSaguiConfig } = saguiConfig
```

Comment:
You shouldn't need to extract `chunks` here. Just let it in the `sharedSaguiConfig`.

Then, you can remove the extra parameter you've add (`chunks`) in a couple of places.

Makes sense?

Reply:
Will fix it